packages/fast-form-generator-react/src/form/form-category.tsx

```diff
@@ -0,0 +1,136 @@
+import * as React from "react";
+import IFormItemCommon from "./form-item";
+import styles from "./form-category.style";
+import { IFormCategoryClassNameContract } from "../class-name-contracts/";
+import manageJss, { IJSSManagerProps } from "@microsoft/fast-jss-manager-react";
+import { IManagedClasses } from "@microsoft/fast-components-class-name-contracts-base";
+
+/**
+ * Select state interface
+ */
+export interface IFormCategoryProps {
+
+    /**
+     * Passes the category item
+     */
+    categoryItem: JSX.Element[];
+
+    /**
+     * Passes the category title
+     */
+    title: string;
+
+    /**
+     * Is the category expandable
+     */
+    Expandable?: boolean;
+
+    /**
+     * Is the category expanded
+     */
+    isExpanded?: boolean;
+
+    /**
+     * Passes the id
+     */
+    id?: string;
+}
+
+export interface IFormCategoryState {
+    isCategoryExpanded: boolean;
+}
+
+/**
+ * Schema form component definition
+ * @extends React.Component
+ */
+/* tslint:disable-next-line */
+class FormCategory extends React.Component<IFormCategoryProps & IManagedClasses<IFormCategoryClassNameContract>, IFormCategoryState> {
+
+    constructor(props: IFormCategoryProps & IManagedClasses<IFormCategoryClassNameContract>) {
+        super(props);
+
+        if (this.props.Expandable) {
+            this.state = {
+                isCategoryExpanded: this.props.isExpanded,
+            };
+        }
+    }
+    /**
+     * Renders the component
+     */
+    public render(): JSX.Element {
+        // Exit if the array is only one item long and that item is null or undefined
+        if (this.props.categoryItem.length < 1 && this.props.categoryItem[0] === null ||
+            this.props.categoryItem.length < 1 && this.props.categoryItem[0] === undefined) {
+            return;
+        }
+
+        return (
+            <div key={this.props.title}>
+                {this.renderHeader()}
+                <div
+                    className={this.getClassNames()}
+                    {...this.generateContainerAttributes()}
+                >
+                    {this.props.categoryItem}
+                </div>
+            </div>
+        );
+    }
+
+    private generateContainerAttributes(): React.HtmlHTMLAttributes<HTMLDivElement> {
+        if (!this.props.Expandable) {
+            return;
+        }
+
+        const attributes: Partial<React.HtmlHTMLAttributes<HTMLDivElement>> = {};
+        attributes["aria-hidden"] = !this.state.isCategoryExpanded;
+        attributes.id = this.props.id;

```

Comment:
it seems odd to me that we would only apply the id if the component is also expandable - maybe just apply the id regardless?